DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/SlewingCoef.h

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef ALICEO2_FT0_SLEWINGCOEF_H_
+#define ALICEO2_FT0_SLEWINGCOEF_H_
+////////////////////////////////////////////////
+// Slewing coefficients for FT0
+//////////////////////////////////////////////
+#include "TGraph.h"
+
+#include <vector>
+#include <array>
+#include <utility>
+
+namespace o2
+{
+namespace ft0
+{
+
+struct SlewingCoef {
+  constexpr static int sNCHANNELS = 208;
+  constexpr static int sNAdc = 2;
+  using Point_t = std::pair<double, double>;                                    // points, x and y
```

Comment:
Totally fine by me, I trust you know best. But just as an idea out of curiosity, would there be any benefit with storing the x and y's in separate arrays to later initialize the TGraph passing these arrays directly as constructor arguments, without looping through the points?

Reply:
This `TGraph::TGraph(Int_t n, const Double_t *x, const Double_t *y)` constructor ? yes, this may work if it is split by two vectors and do something like `TGraph(n, (const Double_t *)vecX.data(),(const Double_t *)vecY.data())`. But struct should be changed

Reply:
since we anyway have variable number of points, `TGraph(n, (const Double_t *)vecX.data(),(const Double_t *)vecY.data())` is better also because it has `memcpy` inside. Ok, will apply this mechanism, thanks @andreasmolander !